Sources/PackageDescription/Resource.swift

```diff
@@ -8,12 +8,27 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+/// A resource to bundle with the Swift package.
+///
+/// If a Swift package declares a Swift tools version of 5.3 or later, it can include resource files.
+/// Similar to source code, Xcode scopes resources to a target. As a result,
+/// you need to put them into the folder that corresponds to the target they belong to.
+/// For example, any resources for the `MyLibrary` target need to reside in `Sources/MyLibrary`.
+/// Use subdirectories to organize your resource files in a way that makes it easy to identify and manage them.
+/// For example, put all resource files into a directory named `Resources`,
+/// resulting in all of your resource files residing at `Sources/MyLibrary/Resources`.
+///
+/// Per default, Xcode handles common resources types for Apple platforms automatically.
```

Comment:
We should rephrase this as "when using the Xcode build system" because that's also possible from SwiftPM (using `--build-system xcode`.